### ISSUE TITLE
Uncompilable output when sorting imports

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#1445]
# Language

Typescript

# Sample input source code

```
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from "react";
import { renderFormFieldRow } from "./component";
// @ts-ignore - asdf
import { FormField } from "@internal/package";
```

# Expected output

```
// @ts-ignore - asdf
import { FormField } from "@internal/package";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from "react";
import { renderFormFieldRow } from "./component";
```

# Actual output (or error message)

```
import { configure, simport { FormField } from "@internal/package";"import { configure, shallow } from "enzyme";eimport Adapter from "enzyme-adapter-react-16";iimport React from "react";
import { renderFormFieldRow } from "./component";
```

**Comment by k2snowman69** [comment]
See #1446

**Comment by k2snowman69** [comment]
Fixed in 1.0.0-beta.4
